### DISCUSSION TITLE
How to set preferred layout?

### DISCUSSION THREAD

**Discussion by Asperamanca** [AnySoftKeyboard/AnySoftKeyboard#3348] [Q&A]
I have two keyboard layouts, the default English one and one German variety. My preferred choice is German, only occasionally do I want to switch to English keyboard.

Multiple times a day, the keyboard will start with English nonetheless.

What I have tried:
- Pulled the German layout to the top in "Keyboards" view (I hoped the top one would be a kind of default)
- Turned off "Remeber alphabet layout per App" (didn't work with "on", either)

Where do I actually set the default layout I want?

**Comment by alchemiker** [comment]
Hi Aperamanca, that's very strange. I also use the German layout almost always and the English and Russian Layout only rarely, and for me it works indeed with "Remember layout per app". Of course it won't work if you open an app for the first time, or maybe it could be lost after an update from the app for example, but I would say that in 95% it works. Maybe it's an interference with some other settings? 🤔 But as far as I know there is no "default" setting for the language.
